src/Illuminate/Auth/Access/Gate.php

```diff
@@ -162,7 +164,7 @@ public function check($ability, $arguments = [])
      *
      * @param  string  $ability
      * @param  array  $arguments
-     * @return callable
+     * @return mixed
```

Comment:
Pear cut in two parts
https://github.com/lucasmichot/framework/commit/7d3a637828c03f29cd987413283375888cc36c2e